mojaloop-simulator/values.yaml

```diff
@@ -119,7 +119,8 @@ ingress:
   # This affects the way your rewrite target will work.
   # For more information see "Breaking changes" here:
   # https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md#0220
-  ingressPathRewriteRegex: (/|$)(.*)
+  # ingressPathRewriteRegex: (/|$)(.*)
+  ingressPathRewriteRegex: ''
```

Comment:
Can we just make this a variable that comes from values file? Looks like this is based on the Ingress Controller version being used and different environments seems to have different versions...

Reply:
It's in the values file. Yes, the appropriate value will depend on the ingress controller being used. My query above was addressing the fact that changing this value will break implementations that haven't specified it. This might be desirable in some cases, but should at least be accompanied with a major `chartVersion` bump.